Form/Extension/DisableCsrfExtension.php

```diff
@@ -101,8 +102,8 @@ public function getExtendedType()
     /**
      * {@inheritdoc}
      */
-    public static function getExtendedTypes()
+    public static function getExtendedTypes(): iterable
     {
-        return array('Symfony\Component\Form\Extension\Core\Type\FormType');
+        return [FormType::class];
```

Comment:
Looks like this needs to be changed to  `return array(FormType::class);` to pass CI.